src/components/pricing/select-plan-new/index.tsx

```diff
@@ -0,0 +1,248 @@
+              tabIndex={0}
+              role="radio"
+              aria-active={checked}
+            >
+              {interval}
+            </button>
+          </li>
+        )
+      })}
+    </ul>
+  )
+}
+
+const DEFAULT_FEATURES = [
+  'Full access to all premium courses and lessons',
+  'RSS course feeds for your favorite podcast app',
+  'Offline viewing',
+  'Commenting and support',
+  'Enhanced Transcripts',
+  'Discord access',
+  'Closed captions for every video',
+]
+
+const PlanFeatures: React.FunctionComponent<{
+  planFeatures: string[]
+}> = ({planFeatures = DEFAULT_FEATURES}) => {
+  const CheckIcon = () => (
+    <svg
+      className="text-blue-500 inline-block flex-shrink-0 mt-1"
+      xmlns="http://www.w3.org/2000/svg"
+      width="16"
+      height="16"
+      viewBox="0 0 16 16"
+    >
+      <path
+        fill="currentColor"
+        d="M6.00266104,15 C5.73789196,15 5.48398777,14.8946854 5.29679603,14.707378 L0.304822855,9.71382936 C0.0452835953,9.46307884 -0.0588050485,9.09175514 0.0325634765,8.74257683 C0.123932001,8.39339851 0.396538625,8.12070585 0.745606774,8.02930849 C1.09467492,7.93791112 1.46588147,8.04203262 1.71655287,8.30165379 L5.86288579,12.4482966 L14.1675324,0.449797837 C14.3666635,0.147033347 14.7141342,-0.0240608575 15.0754425,0.00274388845 C15.4367507,0.0295486344 15.7551884,0.250045268 15.9074918,0.578881992 C16.0597953,0.907718715 16.0220601,1.29328389 15.8088932,1.58632952 L6.82334143,14.5695561 C6.65578773,14.8145513 6.38796837,14.9722925 6.09251656,15 C6.06256472,15 6.03261288,15 6.00266104,15 Z"
+      />
+    </svg>
+  )
+
+  return (
+    <ul>
+      {planFeatures.map((feature: string) => {
+        return (
+          <li className="py-2 font-medium flex" key={slugify(feature)}>
+            <CheckIcon />
+            <span className="ml-2 leading-tight">{feature}</span>
+          </li>
+        )
+      })}
+    </ul>
+  )
+}
+
+const GetAccessButton: React.FunctionComponent<{
+  label: string
+  handleClick: () => void
+}> = ({label, handleClick}) => {
+  return (
+    <button
+      className="mt-8 px-5 py-4 text-center bg-blue-600 text-white font-semibold rounded-md w-full hover:bg-blue-700 transition-all duration-300 ease-in-out transform hover:scale-105"
+      onClick={handleClick}
+      type="button"
+    >
+      {label}
+    </button>
+  )
+}
+
+type SelectPlanProps = {
+  prices: any
+  pricesLoading: boolean
+  defaultInterval?: string
+  defaultQuantity?: number
+  handleClickGetAccess: () => void
+  quantityAvailable: boolean
+}
+
+const SelectPlanNew: React.FunctionComponent<SelectPlanProps> = ({
+  quantityAvailable = true,
+  handleClickGetAccess,
+  defaultInterval = 'annual',
+  defaultQuantity = 1,
+  pricesLoading,
+  prices,
+}) => {
+  const individualPlans = filter(
+    prices,
+    (plan: any) => plan.type === 'individual',
+  )
+  const teamPlan = find(prices, {type: 'team'})
+
+  const annualPlan = get(prices, 'annualPrice', {
```

Comment:
our service throws an error if these don't exist, and the page would be broken so since it's the default we can pass in some info.